dato.config.js

```diff
@@ -77,25 +83,17 @@ function pageToJson (page, i18n) {
       .map(item => omit(item, ['id', 'itemType', 'createdAt', 'updatedAt']))
       .map(transformItem)
   }))
-  const image = page.coverImage
-  let coverImage = null
-
-  if (image !== null) {
-    coverImage = {
-      src: image.imgixHost + image.upload.path,
-      width: image.upload.width,
-      height: image.upload.height,
-    }
-  }
 
-  const seo = page.seo.toMap()
+  const slug = page.slug ? `${page.slug}/` : ''
```

Comment:
what is this for? making sure you have only one trailing `/`? please add a small comment

Reply:
@jbmoelker

Reply:
If a page has a slug, it should end with a trailing slash. Netlify redirects routes without one to a route with a trailing slash. This way, there's always a trailing slash and we don't get different versions in for instance analytics. If a page doesn't have slug, like a home page, then the slug is an empty string.